manager-bundle/src/ContaoManager/Plugin.php

```diff
@@ -314,7 +314,7 @@ private function addDefaultServerVersion(array $extensionConfigs, ContainerBuild
             $connection->connect();
             $connection->query('SHOW TABLES');
             $connection->close();
-        } catch (DriverException $e) {
+        } catch (DriverException | \mysqli_sql_exception $e) {
```

Comment:
@ausi on my local 8.1 test machine (yeah, did it…) this error was triggered when executed by `PluginTest#testDoesNotAddDefaultPdoOptionsIfDriverIsNotPdo()`:

```
mysqli_sql_exception: No database selected
```

While I assume the fix is still correct, the test might be testing the wrong thing? Wdyt?

Reply:
We could add a test that tests this default server version explicitly I guess?
Or do you mean we should check why there is no database selected in the affected test?

Reply:
As discussed on Slack we should maybe even catch `\Throwable` because an exception should never bubble here.

I looked again at the test. I think it's OK the way it is now. The try block will always fail - if we wanted to prevent this we would need to inject a mocked DBAL connection factory or make it functional. IMHO not worth it.